cmd/testrunner.go

```diff
@@ -116,6 +117,11 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return cobraext.FlagParsingError(err, cobraext.ReportOutputFlagName)
 		}
 
+		testCoverage, err := cmd.Flags().GetBool(cobraext.TestCoverageFlagName)
```

Comment:
Should this flag only be allowed if `--report-output file` is also used?

Reply:
The `--report-output` refers to test results, so I assumed it's something else. I don't see a value in printing the coverage to STDOUT, so I didn't bind these flags together.

Do you think we should introduce additional flag for `--coverage-output file` (or stdout) or is there low gain?

Reply:
I was thinking that it may be weird to have files created even when there is only stdout output by default. But improving this, if needed, sounds like something for the future. For example we might show coverage info in the normal test reports, this could be nice, then `--report-output` would control the output of both features.

Reply:
I see, let's keep it for further improvement if there is feature need for this feature.